src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -92,7 +92,7 @@ public function getItemFromIri(string $iri, array $context = [])
     public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH): string
     {
         $resourceClass = $this->getObjectClass($item);
-        $routeName = $this->routeNameResolver->getRouteName($resourceClass, false);
+        $routeName = $this->routeNameResolver->getRouteName($resourceClass, OperationType::ITEM, []);
```

Comment:
Yes, that's right - it functions perfectly well, but logs two deprecation notices. The only effect of the change is that the two deprecations notices are no longer logged.